Mathlib/Combinatorics/Optimization/Deleteme.lean

```diff
@@ -31,7 +32,7 @@ private def exampleFiniteValuedCsp : ValuedCsp ℚ ℚ := {exampleAbs}
 private lemma abs_in : ⟨1, absRat⟩ ∈ exampleFiniteValuedCsp := rfl
 
 private def exampleFiniteValuedInstance : exampleFiniteValuedCsp.Instance (Fin 2) :=
-  [valuedCspTermOfUnary abs_in 0, valuedCspTermOfUnary abs_in 1]
+  Multiset.ofList [valuedCspTermOfUnary abs_in 0, valuedCspTermOfUnary abs_in 1]
```

Comment:
```suggestion
  {valuedCspTermOfUnary abs_in 0, valuedCspTermOfUnary abs_in 1}
```

Reply:
Very nice! Didn't know this notation existed!

Reply:
`{x,y,z}` works for everything that has `Singleton` and `Insert`, so `Set`, `Finset`, `Multiset`, etc.